### DISCUSSION TITLE
Ищу людей которые бы хотели вместе изучать данную модель

### DISCUSSION THREAD

**Discussion by dmkn96** [xai-org/grok-1#250] [Show and tell]
Всем привет кто читает данный пост.

Меня зовут Дима, изучаю ИИ. Если есть желающие русскоговорящие кто бы хотел вместе пытаться 
изучать Грок1 пытаться файнтюнить его и в целом изучать ии совместно, напишите мне)

**Comment by SamKnightV** [comment]
Привет. Мой телеграмм @SamKnightV У меня есть желание разобраться в этом. Напиши, как будет удобно.

**Comment by FoxWerewolf** [comment]
Здравствуй! У меня вопросик по установке, а так в целом почему бы и нет? [@FoxShifter](https://t.me/FoxShifter)

**Reply by yarodevuci** [reply]
@dmkn96 а что тут изучать? у тебя есть обородуывание запустить эту модель вообще?

**Reply by dmkn96** [reply]
Точно не уверен, буду пробовать на видеокарте запускать.

**Comment by yarodevuci** [comment]
эта модель до 2021 года и данные в ней уже устаревшие ....

**Reply by shuen2023** [reply]
Some people say it will end in 2023. Which one is correct?

**Reply by dmkn96** [reply]
I think so, I will answer you after testing.

**Reply by dmkn96** [reply]
I launched the model and fixed all the bugs. 
However, I can't fully run it on my machine 😞 

I've read forums where people mention the need for server-grade hardware. 
At minimum, 100GB of RAM and 5 video cards with more than 8GB of memory each.

⭐ The best option would be to wait for a lighter version with fewer parameters.

**Comment by Mohsin-963** [comment]
Я также изучаю информатику. Пока не русскоязычный человек, который хотел бы изучать ИИ.

**Comment by dmkn96** [comment]
> Some people say it will end in 2023. Which one is correct?
